### DISCUSSION TITLE
Stellarium Web Engine - How to Use?

### DISCUSSION THREAD

**Discussion by WT-MM** [Stellarium/stellarium#3210] [Q&A] [answered]
Hey there, I'm a student hoping to incorporate the Stellarium web engine for a school project. 

I'm not sure if this is the right place to ask, but I couldn't really find any sort of forum for the web engine specifically.

I know nothing about Vue and basically nothing about building web apps, so I've been learning as I go. So far, I've managed to build web-frontend and get that running locally. 

One thing that I figured I should ask about before attempting is the creation of custom sky objects. Is it possible to add custom stars with their own images and descriptions?

Thanks in advance!

**Comment by gzotti** [comment, marked as answer]
We 3 can only discuss the desktop program here. Sorry, we don't provide free tutorials/teaching over what's available in the User Guide. We know as much as there is written documentation.

**Reply by WT-MM** [reply]
Ah, okay

Thanks anyways
